shared/src/vyxal/VList.scala

```diff
@@ -182,12 +182,6 @@ class VList private (val lst: Seq[VAny])
         true
     })
 
-  @targetName("setDiff")
-  def -(other: VList): VList =
```

Comment:
Unrelated to this PR, but it turns out `filterNot` can do the job slightly more concisely and it's only used in one place (credit to Ruben Verg for the change)